packages/server/src/__tests__/errors.test.ts

```diff
@@ -41,36 +41,32 @@ describe('Errors', () => {
     };
 
     it('exposes a stacktrace in debug mode', () => {
-      const error = createFormattedError({
-        includeStackTracesInErrorResponses: true,
-      });
+      const thrown = new Error(message);
+      (thrown as any).key = key;
+      const error = normalizeAndFormatErrors([
+        new GraphQLError(thrown.message, { originalError: thrown }),
+        { includeStacktraceInErrorResponses: true },
+      ])[0];
       expect(error.message).toEqual(message);
-      expect(error.extensions.key).toEqual(key);
-      expect(error.extensions.exception.key).toBeUndefined();
-      expect(error.extensions.code).toEqual(code);
-      // stacktrace should exist under exception
-      expect(error.extensions.exception.stacktrace).toBeDefined();
+      expect(error.extensions?.key).toEqual(key);
```

Comment:
`extensions` are always present, you shouldn't need optional chaining here.

Reply:
@IvanGoncharov It's actually optional on GraphQLFormattedError, which is what this is.